### PR TITLE
man: systemd.unit example text implies clearing list

### PR DESCRIPTION
**Pull request by grooverdan**
"one needs to first clear the list before re-adding all entries except the one that is to be removed. See below for an example." probably should clear the list in the example.

### CODE REVIEW

**Comment by yuwata** [comment]
> "one needs to first clear the list before re-adding all entries except the one that is to be removed. See below for an example."

This note is for drop-in files. Adding `ExecStart=` with empty string in the vendor-supplied unit is meaningless and not suitable for the example.
Also, there already exists an example corresponding to the note. See the following:
https://github.com/systemd/systemd/blob/18a121f9b462e2241c4a590f0a47f5351cd47e0f/man/systemd.unit.xml#L1677-L1678

**Comment by grooverdan** [comment]
moved comment to the example suggested.

File: man/systemd.unit.xml [hunk 1]
```diff
@@ -1540,8 +1533,17 @@ ExecStart=/usr/sbin/some-fancy-httpd-server
...
+
+      <para>Note that for drop-in files, if one wants to remove
+      entries from a setting that is parsed as a list (and is not a
+      dependency), such as <varname>AssertPathExists=</varname> (or
+      e.g. <varname>ExecStart=</varname> in service units), one needs
+      to first clear the list before re-adding all entries except the
+      one that is to be removed. See below for an example.</para>
+
+      <para>Below is a drop-in file for the <emphasis>httpd.service</emphasis>
+      called <filename>/etc/systemd/system/httpd.service.d/local.conf</filename>
```
Comment (yuwata) [inline man/systemd.unit.xml]: 'Below is a drop-in file with the following contents' sounds strange for me.

**Comment by yuwata** [comment]
One more thing; please update the commit message.

**Comment by grooverdan** [comment]
moved note to afterwards and altered commit message.

**Comment by yuwata** [comment]
lgtm. Thanks.

**Pull request merged**